_data/federal_county_production/AL.yml

```diff
@@ -5,46 +5,37 @@
       name: Gas
       units: mcf
       volume:
-        '2007': 66305
         '2008': 61347
         '2009': 60034
         '2010': 48367
         '2011': 43324
         '2012': 41802
-        '2013': 51251
-        '2014': 40642
-        '2015': 23365
-        '2016': 52427
+        '2013': 52715
+        '2014': 40669
+        '2015': 23373
+        '2016': 52449
+        '2017': 45012
     Oil (bbl):
       name: Oil
       units: bbl
       volume:
-        '2007': 8
         '2008': 10
         '2009': 7
         '2010': 24
         '2011': 15
         '2012': 19
-        '2013': 19
+        '2013': 21
         '2014': 16
-        '2015': 9
+        '2015': 10
         '2016': 22
-'01007':
-  name: Bibb County
-  products:
-    Clay (tons):
-      name: Clay
-      units: tons
-      volume:
-        '2007': null
+        '2017': 17
```

Comment:
This is the clay in Bibb County that I don't see on the site or in the spreadsheet.

Reply:
This says it was removed